Sigma_specification.md

```diff
@@ -285,27 +285,38 @@ We recommend using a "definition" value in cases in which further explanation is
 * product - examples: windows, apache, check point fw1
 * service - examples: sshd, applocker
 
-The "category" value is used to select all log files written by a certain group of products, like firewalls or web server logs.
-The automatic converter will use the keyword as a selector for multiple indices.
+The `category` value is used to select all log files written of a logical group. \
+This may cover one or more sources of information depending on the system. \
```

Comment:
Most of the SigmaHQ rules are `process_creation` rules. We should mention this prominent category as example 😉

Reply:
The example below is with process_creation.